pom.xml

```diff
@@ -830,7 +830,7 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>4.12</version>
+                <version>4.11</version>
```

Comment:
Please don't downgrade, but do the following:

- Keep Junit at 4.12
- Upgrade mockito-core to 1.10.19
- Upgrade powermock.version to 1.6.5

Then fix two lines in test 
ninja-maven-plugin/src/test/java/ninja/build/WatchAndRestartMachineTest.java (you'll get exceptions anyway when building).

```
verify(restartTrigger, timeout(10000).never()).trigger(); 
=>
verify(restartTrigger, Mockito.after(10000).never()).trigger();
```

Then we got the latest versions and everything is fine :)